cmd/exp/yaml_migrate.go

```diff
@@ -0,0 +1,19 @@
+package exp
+
+import (
+	"github.com/chanzuckerberg/fogg/exp/yaml_migrate"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	ExpCmd.AddCommand(yamlMigrateCmd)
```

Comment:
`exp` subcommand was intended for experimental commands, this migration should be something we support more broadly. there's already an upgrade command, i wonder if we can latch onto that?